packages/peregrine/lib/talons/CategoryTree/index.js

```diff
@@ -0,0 +1,3 @@
+export { useCategoryBranch } from './useCategoryBranch';
```

Comment:
Once we're done with the peregrine refactors I'd like to have one PR where we go through and create these indexes and exports. Right now we're importing with full paths in `venia-ui` but we can clean all that up with proper exports.